### ISSUE TITLE
Parse Error : syntax error, unexpected '(' on line 20  18 |                     $table->id();  19 |                 $table->bigInteger('galeria_id')->unsigned();> 20 |                 $table->('nome');     |                        ^  21 |                 $table->timestamp('created_at')->nullable();  22 |                 $table->timestamp('updated_at')->nullable();  23 |             

### ISSUE THREAD

**Issue by kinzinho666** [VemtoOrg/vemto2-issues#43]
I followed the instructions, created a new installation using only Breeze and Livewire, connected to Vemto, created the table structure, and clicked on Save.

**Error Message:**
Parse Error : syntax error, unexpected '(' on line 20
  18 |                     $table->id();
  19 |                 $table->bigInteger('galeria_id')->unsigned();
> 20 |                 $table->('nome');
     |                        ^
  21 |                 $table->timestamp('created_at')->nullable();
  22 |                 $table->timestamp('updated_at')->nullable();
  23 |             

**Stack:**
```
SyntaxError: Parse Error : syntax error, unexpected '(' on line 20
  18 |                     $table->id();
  19 |                 $table->bigInteger('galeria_id')->unsigned();
> 20 |                 $table->('nome');
     |                        ^
  21 |                 $table->timestamp('created_at')->nullable();
  22 |                 $table->timestamp('updated_at')->nullable();
  23 |             
    at mt.raiseError (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:271:175)
    at mt.error (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:271:830)
    at mt.read_what (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:270:8537)
    at mt.recursive_variable_chain_scan (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:270:9357)
    at mt.read_variable (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:270:7207)
    at mt.read_expr_item (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:19098)
    at mt.read_expr (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:260:12038)
    at mt.read_statement (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:270:1944)
    at mt.read_inner_statement (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:267:3594)
    at mt.read_inner_statements (file:///C:/Users/k1/AppData/Local/Programs/vemto/resources/app.asar/renderer/assets/index.a93dd9ee.js:267:2090)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

![estrutura](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/29714101/7850c006-b3ab-4048-9e7e-62a0f9ed2fa7)

Vemto 2.0.5
win32 10.0.22631
Locale: pt-BR
App version: 2.0.5

**Comment by kinzinho666** [comment]
i found error, field NOME on kartodromos tabel is empty, you can show alert when type is empty

**Comment by TiagoSilvaPereira** [comment]
I'll reopen as we need to fix this bug

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7
